client/src/Pages/MentorsEdit.js

```diff
@@ -15,27 +15,27 @@ export default function MentorsEdit() {
  
 return(
     <Router>
-      <div>
-        <h2>Skills</h2>
+      <div className="skills-container">
+        <h2>Skill</h2>
 
         <ul>
           <li>
-            <Link to="/mentorsedit/html">HTML</Link>
+           <h2><Link to="/mentorsedit/html">HTML</Link></h2> 
```

Comment:
`<h2>` and all other heading elements are supposed to provide a hierarchy to your page content. I don't really think that these links are headings, so I would personally avoid putting them in a `<h*>` tag.

If you want to style the links (make them bolder/bigger/different colour etc.) I recommend you use a CSS class (`className` in React JSX) instead.